### ISSUE TITLE
Access for Lenae Storey

### ISSUE THREAD

**Issue by lenaecb** [department-of-veterans-affairs/va.gov-team#8197]
# Instructions
1. Fill out **Description**
2. Add the label used by your team (eg: `BAH-526`)
3. Add a comment @department-of-veterans-affairs/vsp-operations to ping the operations team
4. When the issue is closed you will be notified and can continue with on-boarding setup

# Description
- Please provide your Contracting Officer's Representative (COR) name.
   > Eric Hopkins
- Please provide your name.
   > Lenae Storey
- Please Provide your email.
   > lenae.storey@adhocteam.us
- Please provide the name of your team, your role on that team, and the name of the company you work for.
   > VA Online Scheduling on VA.gov, Product Manager, and Ad Hoc
- [if requesting SSH key authorization] What is the **public** SSH key? - Yes requesting authorization. Key below.
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCdfNxBr0LNy8JssDuwJA3EhAyTBiOysdiIvJDAxwnesENTS5sVCGI6EvlC1bTVbisQ+hWlJDdwD+h21Zl4e8DO7/EBSOh7D8a+bj+rY6GC0d6A9czfrfk2K22sydUi4HV073mSgr8Ypv1kC03D6FY/kqgXTd/Tkz4xc8V0XdvW8JlExoInFnkmBrKb+RQDWL8xHLTDJTa/YvReD0KfninWRbd44MtjoIY6CUAN2wq8uzXEoSyySrL1dP1eT16xjeam7nGXb9Ryc1ecEVcEjACxJr31MFNMlzPgioPiG/JL+7TvM2ASwVHE/v/sWbb5wH5sgnmsxLwkmBoPrnphY4gz lenaestorey@Lenae-Storeys-MacBook-Pro.local

- [Grant AWS Console access?]
  > No
  >  Please give a brief description for why you are requesting AWS access.

**Comment by jbritt1** [comment]
SOCKS access granted. Please allow up to ten minutes for the new configuration to deploy and continue with [the guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/internal-tools.md#configure-the-socks-proxy) to establish a connection to SOCKS proxy.

**Comment by lenaecb** [comment]
@jbritt1 - thanks. I am having some issues getting the proxy connected. Specifically for Sentry. 

I am getting this error from SwitchyOmega

![Screen Shot 2020-04-21 at 10 40 18 AM](https://user-images.githubusercontent.com/16602559/79879315-8532a300-83bc-11ea-9a9d-5eea847e5e33.png)

I have checked my proxy settings on both my computer and in Chrome. Thoughts?

**Comment by lenaecb** [comment]
Here are my SwithcyOmega settings...

![Screen Shot 2020-04-21 at 10 41 04 AM](https://user-images.githubusercontent.com/16602559/79879440-a98e7f80-83bc-11ea-94b3-be51762a16a2.png)
![Screen Shot 2020-04-21 at 10 41 11 AM](https://user-images.githubusercontent.com/16602559/79879444-ab584300-83bc-11ea-87cd-43b6f3c50955.png)

**Comment by jbritt1** [comment]
Did you ensure to set "auto-switch" on the plugin like below?

![image](https://user-images.githubusercontent.com/56260532/79879726-1570e800-83bd-11ea-89da-640e99d5d9a3.png)

**Comment by jbritt1** [comment]
I say that because `*.va.gov` should not actually be using the SOCKS proxy. May I also ask what URL you are trying to hit when receiving that error?

**Comment by lenaecb** [comment]
Yea, it's enabled. And I also thought it was strange that it's trying to apply the *.va.gov when the setting is *.vfs.va.gov... 
![Screen Shot 2020-04-21 at 10 45 16 AM](https://user-images.githubusercontent.com/16602559/79879924-5406a280-83bd-11ea-8d75-4f224b5a86a2.png)

Trying to go to - http://sentry.vfs.va.gov/

**Comment by jbritt1** [comment]
Alright, let's hop over to Slack. Are you in the DSVA workspace yet? Do you have time for a potential call / screenshare?

**Comment by lenaecb** [comment]
@jbritt1 - yes to both

**Comment by jbritt1** [comment]
Alright, messaging you there momentarily. Reopened ticket until resolved for tracking.

**Comment by jbritt1** [comment]
Brief synopsis:

So a few things happened here:

 - Lenae's pc shutdown yesterday for some updates we think and her `~/.ssh/config` was lost. We discovered and resolved.
 - Because of the pc restart, the terminal had closed so her key was no longer added to the ssh-agent. She now knows to ensure her key is added with `ssh-add ~/.ssh/id_rsa_vetsgov`
 - Her connection to SOCKS needed to be ran the first time without the `&` to accept the initial ssh connections, adding them to the known hosts of the machine.
- Lenae now has a working SOCKS proxy and a note on necessary commands to run in order to get it working again going forward. 

Resolved at this time.
